### PR TITLE
Update upgrade.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed reference to RHEL 6 and Ubuntu 16.04.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@svelle, let me know what the commands are for the latest versions and I can add them. :)

**Comment by svelle** [comment]
> @svelle, let me know what the commands are for the latest versions and I can add them. :)

Hey  @justinegeffen you can remove the `On Ubuntu 16.04 and RHEL 6` and just leave the rest as the command should work on every distro since then.

File: source/administration/upgrade.rst [hunk 1]
```diff
@@ -123,12 +115,6 @@ Location of your local storage directory
 
 #. Start your Mattermost server.
 
-   On Ubuntu 14.04 and RHEL 6:
-
-   .. code-block:: sh
-
-     sudo service mattermost start
-
    On Ubuntu 16.04 and RHEL 7:
```
Comment (svelle) [inline source/administration/upgrade.rst:118]:
```suggestion
```

File: source/administration/upgrade.rst [hunk 2]
```diff
@@ -62,12 +60,6 @@ Location of your local storage directory
 
 #. Stop your Mattermost server.
 
-   On Ubuntu 14.04 and RHEL 6:
-
-   .. code-block:: sh
-
-     sudo service mattermost stop
-
    On Ubuntu 16.04 and RHEL 7:
```
Comment (svelle) [inline source/administration/upgrade.rst:63]:
```suggestion
```

**Review by svelle — CHANGES_REQUESTED** [review]
If we remove this I think we should be good :)

**Comment by justinegeffen** [comment]
@jasonblais, adding you for editorial review as Amy is on PTO. :)

**Comment by justinegeffen** [comment]
@svelle, just confirming that we're good to go with this. :)

**Comment by svelle** [comment]
> @svelle, just confirming that we're good to go with this. :)

Good to go from my end!

**Pull request merged**